models/DeliveryExecutionFinderService.php

```diff
@@ -39,23 +36,19 @@ class DeliveryExecutionFinderService extends ConfigurableService
 {
     public const SERVICE_ID = 'taoReview/DeliveryExecutionFinderService';
 
-    protected const LTI_SOURCE_ID = 'lis_result_sourcedid';
+    public const LTI_SOURCE_ID = 'lis_result_sourcedid';
 
     /**
      * @param LtiLaunchData $data
      *
      * @return DeliveryExecution
-     * @throws InvalidServiceManagerException
      * @throws LtiInvalidLaunchDataException
      * @throws LtiVariableMissingException
      */
     public function findDeliveryExecution(LtiLaunchData $data): DeliveryExecution
```

Comment:
Let's find a more appropriate variable name, `data` is the worst choice, it means nothing.